src/OpenConext/ProfileBundle/Resources/views/MyServices/ConsentList/service-information.html.twig

```diff
@@ -2,13 +2,29 @@
 {% if specifiedConsent.releasedAttributes is empty %}
     <p>{{ 'profile.my_services.no_attribute_released'|trans }}</p>
 {% else %}
-    {% for sourceName, releasedAttributes in specifiedConsent.releasedAttributesGroupedBySource  %}
-        {# TODO: the AA sourced attributes are yet to be rendered see: https://www.pivotaltracker.com/story/show/179920711 for details #}
+    {% for sourceName, releasedAttributes in specifiedConsent.getIdPAttributes  %}
         {% if sourceName == 'idp' %}
         <ul class="attributeList">
             {% include '@OpenConextProfile/MyServices/AttributeList/idp.html.twig' %}
         </ul>
         {% endif %}
     {% endfor %}
+
+    {% if specifiedConsent.getAttributeAggregatedAttributes|length > 1 %}
+
+    <p>{{ 'profile.my_services.service_information.aa_text'|trans }}</p>
+
+    <ul class="attributeList">
+    {% for sourceName, releasedAttributes in specifiedConsent.getAttributeAggregatedAttributes  %}
+        {% include '@OpenConextProfile/MyServices/AttributeList/aa.html.twig' %}
+    {% endfor %}
+    <li class="aaRow">
+        {% set singular = 'profile.table.explanation.singulare'|trans %}
+        {% set plural = 'profile.table.explanation.plural'|trans %}
+        {% set singularOrPlural = specifiedConsent.getAttributeAggregatedAttributes|length > 1 ? plural : singular %}
```

Comment:
Not sure if this now fetches it for the third time and it would be more efficienty to fetch it once and then check the count + iterate over it+ check the count here? Might be premature optimization though

Reply:
For now I'm leaving this as-is